### PR TITLE
Add SLO RBAC information

### PR DESCRIPTION
**Pull request by estherk15**
### What does this PR do?
- Add information to SLO page regarding SLO RBAC and restricting individual SLO access
- Add image of the SLO Permissions option
- Add SLO to the RBAC list of resources that restrict access

### Motivation
[DOCS-4639](https://datadoghq.atlassian.net/browse/DOCS-4639)

### Additional Notes
- Need AAA review
- Merge docs after API is available to all users

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

[DOCS-4639]: https://datadoghq.atlassian.net/browse/DOCS-4639?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: content/en/monitors/service_level_objectives/_index.md [hunk 1]
```diff
@@ -159,7 +178,7 @@ Three types of SLO audit events appear in the Event Explorer:
 
 To get a full list of all SLO audit events, enter the search query `tags:audit,slo` in the Event Explorer. To view the list of audit events for a specific SLO, enter `tags:audit,slo_id:<SLO ID>` with the ID of the desired SLO.
 
-You can also query the Event Explorer programmatically using the [Datadog Events API][13].
+You can also query the Event Explorer programmatically using the [Datadog Events API][17].
```
Comment (rtrieu) [inline content/en/monitors/service_level_objectives/_index.md:181]: This link didn't work for me (led to a 404).

**Review by rtrieu — CHANGES_REQUESTED** [review]
Maybe it is just me, but there is a link that led to a 404. Other than that, the update is great!

File: content/en/monitors/service_level_objectives/_index.md (hunk 1 above)
Comment (estherk15) [reply content/en/monitors/service_level_objectives/_index.md:181]: Thank you for catching this!

**Review by rtrieu — APPROVED** [review]

File: content/en/monitors/service_level_objectives/_index.md [hunk 2]
```diff
@@ -77,6 +76,34 @@ Setting a 100% target means having an error budget of 0% since error budget is e
 
 To edit an SLO, hover over the SLO's row in the list view and click the edit pencil icon that appears at the right of the row, or click on the row to open the details side panel and select the edit button from the cog icon in the top right of the panel.
 
+## Permissions
+
+#### Role based access
```
Comment (kaixintai) [inline content/en/monitors/service_level_objectives/_index.md:81]: Capability based access controls

File: content/en/monitors/service_level_objectives/_index.md [hunk 3]
```diff
@@ -77,6 +76,34 @@ Setting a 100% target means having an error budget of 0% since error budget is e
...
+
+All users can view SLOs and [SLO status corrections](#slo-status-corrections), regardless of their associated [role][9]. Only users attached to roles with the `slos_write` permission can create, edit, and delete SLOs.
+
+To create, edit, and delete status corrections, users require the `slos_corrections` permissions. A user with this permission can make status corrections, even if they do not have permission to edit those SLOs. For the full list of permissions, see the [RBAC documentation][10].
+
+#### Restrict SLO permissions to specific roles
```
[the remark this answers is not in the document]
Comment (kaixintai) [reply content/en/monitors/service_level_objectives/_index.md:87]: Also, we should add a link for SLOs to the list here https://docs.datadoghq.com/account_management/rbac/?tab=datadogapplication#restrict-access-to-individual-resources

File: content/en/monitors/service_level_objectives/_index.md (hunk 2 above)
Comment (estherk15) [reply content/en/monitors/service_level_objectives/_index.md:81]: @kaixintai I don't see this used in any other product for Permissions features, let me know if I missed this somewhere. Is this specific to SLOs?

Comment (kaixintai) [reply content/en/monitors/service_level_objectives/_index.md:81]: It was the terminology I was advised to use by @sidph. Sid, is this ok or is there something else you'd like to call resource RBAC?

Comment (kaixintai) [reply content/en/monitors/service_level_objectives/_index.md:81]: Also Esther, we're blocked by the production freeze, so we haven't enabled granular RBAC for customers yet. Will plan to do it next week and ping you when it's on.

**Review by kaixintai — APPROVED** [review]
LGTM! We've enabled for US3 and 5. Going to enable for US1 and EU later this week.

**Pull request merged**